resources/sass/original.scss

```diff
@@ -1,27 +1,30 @@
 @import "base";
 
+:root {
+  --kint-color1: #e3ecf0;
```

Comment:
Let's keep these as variables. My reasoning is that when we need to introduce the dark edition of this theme, we need the vars so that we could override them.

If we don't use variables here now, in the next PR those needs to be added later anyway.

Reply:
Good point... Then just leave them unnamed for now, I'll think of something to call them later